### PR TITLE
changes

### PR DESCRIPTION
**Pull request by otimebhor**

### CODE REVIEW

File: src/create-accounts/create-accounts.service.ts [hunk 1]
```diff
@@ -323,32 +331,40 @@ export class CreateAccountsService {
...
-    }, 2 * 24 * 60 * 60 * 1000); 
+    setTimeout(
+      async () => {
+        await this.userRepository.update(user.id, { password: null });
+      },
+      2 * 24 * 60 * 60 * 1000
+    );
 
-  
     const loginUrl = `${process.env.FRONTEND_URL}/auth/login}`;
```
Comment (Tes-program) [inline src/create-accounts/create-accounts.service.ts:348]: The frontend url is https://frontend-mvp-gt99.vercel.app/signIn. Why is yours different?

File: src/create-accounts/create-accounts.service.ts [hunk 2]
```diff
@@ -256,14 +258,14 @@ export class CreateAccountsService {
     });
 
     await this.studentRepository.save(student);
-    
+
     const loginUrl = `${process.env.FRONTEND_URL}/auth/login}`;
```
Comment (Tes-program) [inline src/create-accounts/create-accounts.service.ts:262]: Same issue here.

File: src/template/welcome-users.html
Comment (Tes-program) [inline src/template/welcome-users.html:1]: Why is the title Onboarding Mail? Is that the title of this mail? Also this mailing template is old shaa and not similar to what we have on figma. You can check my template and copy the code shaa.

**Review by Tes-program — CHANGES_REQUESTED** [review]

File: src/template/welcome-users.html
Comment (otimebhor) [reply src/template/welcome-users.html:1]: This is the template @Mbaoma shared on confluence.

File: src/create-accounts/create-accounts.service.ts (hunk 1 above)
Comment (otimebhor) [reply src/create-accounts/create-accounts.service.ts:348]: I will change it

**Review by Mbaoma — APPROVED** [review]

**Review by Tes-program — APPROVED** [review]

**Pull request merged**